wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -9,6 +9,9 @@
     - `description` - if present, displayed as a small text below the `h1` tag title
     - `search_url` - if present, display a search box. This is a URL route name (taking no parameters) to be used as the action for that search box
     - `search_form` - form object for the search form. Required if search_url is passed
+    - `search_results_url` - URL to be used for async requests to search results, if not provided, the form's action URL will be used
+    - `search_target` - A selector string to be used as the target for teh search results to be swapped into. Defaults to '#results'
+    - `use_alt_search` - Tf true, use the default header async search functionality will not used
```

Comment:
drive-by comment: I am not entirely sure I understand this. As far as I can see if `use_alt_search=True` will resulting in using AJAX when search_results_url is set? if so, should this perhaps be more explicit in naming? 🤔

Reply:
That's not the case sorry, there are a few scenarios where we don't need the explicit search results url as some Async forms use the form action attributes and some don't. 

The longer term goal will be to move the chooser header searches to SwapController so this Param is hopefully short lived.

Reply:
ah! that makes loads more sense. in which case the help text could say something like:
"If true, the default header async search functionality will not be used"?

Reply:
Yeah. Thank you.